packages/features/bookings/lib/handleNewBooking/test/fresh-booking.test.ts

```diff
@@ -3280,6 +3280,126 @@ describe("handleNewBooking", () => {
     });
   });
 
+  describe("Returning original booking", () => {
```

Comment:
nit: I think we could introduce a new module for complete different blocks. This file is huge already.

Something like fresh-booking-unconfimed.test.ts